SOURCES/GAMEMENU.C

```diff
@@ -1226,13 +1350,71 @@ WORD	FindPlayerFile()
 
 /*──────────────────────────────────────────────────────────────────────────*/
 
-void	SaveGame()
+void SaveGame()
+{
+	strcpy(PlayerName, AUTO_SAVE_NAME);
+	SaveGameWithName(AUTO_SAVE_NAME, 1);
+}
+
+void SaveGameWithName(char* fileName, WORD isAutoSave)
 {
 	FILE*	handle ;
 	WORD	wword ;
 	UBYTE	wbyte ;
+	int i;
+
+	if (fileName)
+	{
+		char savePath[1024];
+#ifdef WINDOWS_SAVE
```

Comment:
Perhaps this could be an option on lba.cfg file? So you can toggle the behaviour without recompiling.

Reply:
It could indeed, I will consider this as an enhancement feature :)

Reply:
Implemented. More info here: https://github.com/rmm17/lba1-classic-community/issues/29